activestorage/lib/active_storage/attached/model.rb

```diff
@@ -217,6 +219,12 @@ def validate_service_configuration(association_name, service)
             end
           end
         end
+
+        def validate_global_service_configuration
```

Comment:
Nice, I like this solution much better 👍 

Maybe it would be better if `validate_service_configuration` called this.

Reply:
Yes, I agree it would be better, I had not seen this before.

I've changed it so that if a service is not passed as an argument, then active storage should validate the global service configuration. Here's how I've changed the `validate_service_configuration` method:

https://github.com/rails/rails/blob/516c994c03ae1daff539a2b40f439dbc194ae4cd/activestorage/lib/active_storage/attached/model.rb#L213-L221

@ghiculescu Does this look good?